nss-evening-cohort-7/dating-site-instructorz
# Added Navbar

To test: 
- Pull down code
- Navbar should show up on homepage
- Click on each link and verify it brings you to the correct page via the url
